vspc/server.py

```diff
@@ -84,29 +84,38 @@ class VspcServer(object):
     def __init__(self):
         self.sock_to_uuid = dict()
 
-    async def handle_known_suboptions(self, writer, data):
+    async def handle_known_suboptions(self, writer, data, vm_uuid):
         socket = writer.get_extra_info('socket')
         peer = socket.getpeername()
-        LOG.debug("<< %s KNOWN-SUBOPTIONS-1 %s", peer, data)
-        LOG.debug(">> %s KNOWN-SUBOPTIONS-2 %s", peer, SUPPORTED_OPTS)
+        LOG.debug(
+            "VM-VC-UUID: %s << %s KNOWN-SUBOPTIONS-1 %s",
+            vm_uuid, peer, data
+        )
+        LOG.debug(
+            "VM-VC-UUID: %s >> %s KNOWN-SUBOPTIONS-2 %s",
+            vm_uuid, peer, SUPPORTED_OPTS
+        )
         writer.write(IAC + SB + VMWARE_EXT + KNOWN_SUBOPTIONS_2 +
                      SUPPORTED_OPTS + IAC + SE)
-        LOG.debug(">> %s GET-VM-VC-UUID", peer)
+        LOG.debug("VM-VC-UUID: %s >> %s GET-VM-VC-UUID", vm_uuid, peer)
```

Comment:
I'd ditch the `VM-VC-UUID` and just put an `[%s]` or even '"%s >>"` in there. We don't need those extra characters.
Can we maybe even put it into the already existing `[]`? Where's the log format defined?
> 2020-12-15 06:27:50.723 8 DEBUG vspc.server [-] >> ('10.246.201.66', 34876) DONT handle_will /usr/local/lib/python3.5/site-packages/vspc/server.py:170

Reply:
I think this section `[-]` refers to `Trace level` logging and maybe it can be changed somehow via CONF which passed to `oslo.log`... The simpler way it's using just `[%s]`.